daemon/algod/api/server/v2/generated/data/data_routes.yml

```diff
@@ -5,13 +5,15 @@ generate:
 output-options:
   include-tags:
   - data
-  - public
+# do not include endpoints tagged public/experimental
+# - public
```

Comment:
This works, but it's a bit hacky because the tags weren't meant to be overlapping like this--each set of APIs, gated by certain configs, is tagged as `public` or `private` to identify the authenticator required, and a single category.

This will introduce partially overlapping sets of tags in ways that could become messy in the future. But it does look like the only viable way we currently have to disable sdk code gen for an API w/o requiring the admin token.